src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -148,6 +148,13 @@ public String createSnapshot(
     String description = "Create snapshot " + snapshotRequestModel.getName();
     String sourceDatasetName = snapshotRequestModel.getContents().get(0).getDatasetName();
     Dataset dataset = datasetService.retrieveByName(sourceDatasetName);
+    if (snapshotRequestModel.getProfileId() == null) {
+      snapshotRequestModel.setProfileId(dataset.getDefaultProfileId());
+      logger.warn(
+          String.format(
```

Comment:
FWIW, I don't think you need to use String.format in a warning, I think you can just have the string look like:
`"Enriching {} snapshot {} request with dataset default profileId {}"`

Reply:
Fair enough.  How about at line 657?  Update there too?

Reply:
ooh good call.  Yeah, if you don't mind

Reply:
Done.